xbox/webapi/scripts/authenticate.py

```diff
@@ -1,126 +1,48 @@
 """
 Example scripts that performs XBL authentication
 """
-import sys
-import argparse
+import asyncio
+import os
+import webbrowser
 
-import getpass
+from aiohttp import web, ClientSession
 
 from xbox.webapi.authentication.manager import AuthenticationManager
-from xbox.webapi.authentication.two_factor import TwoFactorAuthentication, TwoFactorAuthMethods
-from xbox.webapi.common.exceptions import AuthenticationException, TwoFactorAuthRequired
-from xbox.webapi.scripts import TOKENS_FILE
 
+CLIENT_ID = os.environ["CLIENT_ID"]
```

Comment:
I agree, passing it via environment variables is what you usually do in production software when it comes to passing such configuration parameters.
Does it work the same in windows tho?

Like: `CLIENT_ID="123" CLIENT_SECRET="321" xbox-authenticate` ?

Reply:
You can't do all-in-one in Windows, but can use `setx CLIENT_ID "12368654"` and it will be available in subsequent command prompts.  Of course there is [also a GUI way](https://www.twilio.com/blog/2017/01/how-to-set-environment-variables.html) of doing it.

Reply:
I would prefer reintroducing argparse then.

Reply:
I can do both. Default env, override argparse.

In VSCode I use a .env file and running the script automatically loads it for me when testing, so it's more convenient than having to keep typing it or copy/paste.